software/base/pom.xml

```diff
@@ -116,7 +116,6 @@
             <groupId>org.apache.brooklyn</groupId>
             <artifactId>brooklyn-locations-jclouds</artifactId>
             <version>${project.version}</version>
-            <scope>test</scope>
```

Comment:
I need a non-test dependency on `brooklyn-locations-jclouds`. Tests also need code from `brooklyn-locations-jclouds` so adding a simple dependency with default compile scope is enough I think?
@neykov WDYT?